src/models/gamma_poisson_model.py

```diff
@@ -113,14 +113,19 @@ class GammaPoissonModel(ReachCurve):
     found.
     """
 
-    def __init__(self, data: List[ReachPoint], max_reach=None):
+    def __init__(
+        self, data: List[ReachPoint], max_reach=None, extrapolation_multiplier=1.0
+    ):
         """Constructs a Gamma-Poisson model of underreported count data.
 
         Args:
           data:  A list of consisting of a single ReachPoint to which the model is
             to be fit.
           max_reach:  Optional.  If specified, the maximum possible reach that can
             be achieved.
+          extrapolation_multiplier:  Int.  If specified, then a penalty term is
+            introduced that penalizes models where the expected number of impressions
```

Comment:
expected number of impressions in the inventory is less than the multiplier times the observed number number impressions, i.e., ....

Reply:
I updated the doc.  Thanks for the suggestion!

Reply:
Awesome update. Thanks!